pkg/subctl/cmd/join.go

```diff
@@ -185,12 +189,63 @@ func joinSubmarinerCluster(config *rest.Config, subctlData *datafile.SubctlData)
 	clusterCIDR, err = getPodCIDR(clusterCIDR, networkDetails)
 	exitOnError("Error determining the pod CIDR", err)
 
+	status.Start("* Discovering multi cluster details")
```

Comment:
nit: multi-cluster

Reply:
Please remove the "* " at the start, this is handled by the status library.